adapters/zwave/zwave-classifier.js

```diff
@@ -46,7 +47,28 @@ class ZWaveClassifier {
             'name': 'toggle',
             'description': 'Toggles On/Off',
         }];
-        node.propertyMap = {'on': valueId };
+        node.propertyMaps = {'on': valueId };
+        node.values.on = node.zwValues[valueId].value;
+
+        if (node.zwInfo.productId == 0x0060 &&
+            (node.zwInfo.productType & 0xff) == 0x03) {
```

Comment:
This is ringing some alarm bells for me. Do we have to write code like this for every model of Z-Wave smart plug on the market? Is there a more generic way to do this that might work for multiple smart plugs?